stories/core/Surface.stories.tsx

```diff
@@ -0,0 +1,43 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import { Story, Meta } from '@storybook/react';
+import { CreeveyMeta } from 'creevey';
+import React from 'react';
+import { Surface, SurfaceProps } from '../../src/core';
+
+export default {
+  component: Surface,
+  argTypes: {
+    className: { control: { disable: true } },
+    style: { control: { disable: true } },
+    id: { control: { disable: true } },
+    elevation: {
+      options: [0, 1, 2, 3, 4, 5],
+    },
+  },
+  parameters: {
+    creevey: {
+      captureElement: null,
+    },
+  },
+  title: 'Core/Surface',
+} as Meta<SurfaceProps> & CreeveyMeta;
+
+export const Basic: Story<SurfaceProps> = ({ elevation = 0 }) => {
```

Comment:
Can you change the story elevation to 1 so that the initial view of the story has a shadow?

Reply:
That's a good solution. This is pretty nit-picky and not necessary, but if you add padding to the cardStyle the text won't be right against the edge of the box.